src/react/components/MnemonicGenerationWizard.tsx

```diff
@@ -124,15 +91,19 @@ const MnemonicGenerationWizard: FC<Props> = (props): ReactElement => {
       props.onStepForward();
     } else {
       setMnemonicValidationError(true);
+      setStep(step-1); // back to 3
     }
   }
 
   const uiCreateMnemonic = () => {
-    if (uname() == "Linux") {
-      props.setMnemonic(createMnemonic('english'));
-    } else {
-      props.setMnemonic("one two three four five six seven eight nine ten eleven twelve one two three four five six seven eight nine ten eleven twelve");
-    }
+    console.log("Generating mnemonic...");
+    createMnemonic('english')
+      .then((mnemonic) => {
+        props.setMnemonic(mnemonic);
+      })
+      .catch((error) => {
+        console.log(error);
```

Comment:
Eventually, I think this should be showed to the user somehow. Could be in another PR or tracked in an issue.

Reply:
Great call.  Added #62.